### PR TITLE
Switch to OpenTK GameWindow instead of WinForms+GLControl

### PR DESCRIPTION
**Pull request by SirCmpwn**
Tested, works with both visual tests and osu! on Linux. Fixes weird bug on Linux where several rows of pixels on the top were black. Requires testing on Windows, since all input was switched to OpenTK and I hear that's wonky on Windows.

### CODE REVIEW

**Comment by SirCmpwn** [comment]
TODO: Add SDL native libs for Window's sake.

File: osu.Framework.Desktop/Platform/DesktopGameWindow.cs [hunk 1]
```diff
@@ -40,27 +27,14 @@ internal DesktopGameWindow(GraphicsContextFlags flags)
             Form.SizeChanged += delegate { OnClientSizeChanged(); };
             Form.Paint += delegate { OnPaint(); };
             Form.FormClosing += Form_FormClosing;
-            Form.FormClosed += delegate { OnExited(); };
+            Form.FormClosed += delegate { OnExited(); };*/
```
Comment (SirCmpwn) [inline osu.Framework.Desktop/Platform/DesktopGameWindow.cs]: Don't comment out code

File: osu.Framework.Desktop/Platform/DesktopGameWindow.cs [hunk 2]
```diff
@@ -40,27 +27,14 @@ internal DesktopGameWindow(GraphicsContextFlags flags)
...
-            });
-        }
-
-        protected override void SetTitle(string title)
-        {
-            Form.Text = title;
-        }
-
-        internal TextInputSource CreateTextInput() => new BackingTextBox(Form);
+        internal TextInputSource CreateTextInput() => null;// new BackingTextBox(Form);
```
Comment (SirCmpwn) [inline osu.Framework.Desktop/Platform/DesktopGameWindow.cs]: Don't comment out code

File: osu.Framework/Input/InputManager.cs [hunk 3]
```diff
@@ -601,6 +601,13 @@ private bool handleKeyUp(InputState state, Key key)
             return keyboardInputQueue.Any(target => target.TriggerKeyUp(state, args));
         }
 
+        private bool handleCharacterInput(char c)
+        {
+            if (FocusedDrawable?.TriggerCharacterInput(c) ?? false)
+                return true;
+            return false;
+        }
```
Comment (SirCmpwn) [inline osu.Framework/Input/InputManager.cs]:
```c#
return FocusedDrawable?.TriggerCharacterInput(c) ?? false
```

Should update rest of this file to this pattern as well

**Comment by Tom94** [comment]
A note from my end: We need to _make sure_ that exclusive fullscreen works with this on windows machines with _intel integrated graphics_. This should be the case if GameWindow has its OpenGL content as a top-level form, but not if it uses any level of nesting.

**Pull request merged**